### PR TITLE
docs: user guide about static routing

### PR DESCRIPTION
**Pull request by ShiKaiWi**
# Which issue does this PR close?

Close #

# Rationale for this change
Users may want to deploy CeresDB as a cluster but now there is no such a guide to explain how to create CeresDB cluster.

# What changes are included in this PR?
Provide a guide for how to create static, rule-based routing deployment of CeresDB.

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: docs/user-guide/src/deploy/README.md [hunk 1]
```diff
@@ -1 +1,4 @@
 # Deployment
+CeresDB is a distributed timeseries database, that is to say, multiple CeresDB instances can be deployed as a cluster to serve with high availability and scalability.
+
+For now, CeresDB only supports one deployment form that features static, rule-based routing. In the future (maybe in 1.0.0 version), a powerful cluster deployment will be supported and then CeresDB cluster will support more greate features, including dynamic expansion, data reliability and etc.
```
Comment (waynexia) [inline docs/user-guide/src/deploy/README.md:4]:
```suggestion
For now, CeresDB only supports one deployment form that features static, rule-based routing. In the future (maybe in 1.0.0 version), a powerful cluster deployment will be supported and then CeresDB cluster will support more great features, including dynamic expansion, data reliability and etc.
```

File: docs/user-guide/src/deploy/static_routing.md [hunk 2]
```diff
@@ -1 +1,228 @@
 # Static Routing
+This guide shows how to deloy a CeresDB cluster with static, rule-based routing.
```
Comment (waynexia) [inline docs/user-guide/src/deploy/static_routing.md:2]:
```suggestion
This guide shows how to deploy a CeresDB cluster with static, rule-based routing.
```

File: docs/user-guide/src/deploy/static_routing.md [hunk 3]
```diff
@@ -1 +1,228 @@
...
+wal_path = "/tmp/ceresdb"
+
+[analytic.storage]
+type = "Local"
+data_path = "/tmp/ceresdb"
+```
+
+In order to deploy two CeresDB instances on the same machine, the config should choose different ports to serve and data directories to store data.
+
+Say the CeresDB_0's config is:
```
Comment (chunshao90) [inline docs/user-guide/src/deploy/static_routing.md:28]: The CeresDB_0's config is:

File: docs/user-guide/src/deploy/static_routing.md [hunk 4]
```diff
@@ -1 +1,228 @@
 # Static Routing
+This guide shows how to deloy a CeresDB cluster with static, rule-based routing.
+
+The crucial point here is that CeresDB server provides configurable routing function on table name so what we need is just a valid config containing routing rules which will be shipped to every CeresDB instance in the cluster.
+
+## Target
+First of all, let's assume that our target is to deploy a cluster consisting of two CeresDB instances on the same machine. And a large cluster of more CeresDB instances can deployed according to the two-instance example.
+
+## Prepare Config
```
Comment (chunshao90) [inline docs/user-guide/src/deploy/static_routing.md:9]: split into some sub part

File: docs/user-guide/src/deploy/static_routing.md (hunk 3 above)
Comment (ShiKaiWi) [reply docs/user-guide/src/deploy/static_routing.md:28]: Here `Say` means `Assume`.

**Review by waynexia — APPROVED** [review]

**Pull request merged**